shared/common-adapters/avatar.js

```diff
@@ -156,6 +165,7 @@ const _sharedAskForUserData = new SharedAskForUserData()
 
 const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
   const name = ownProps.username || ownProps.teamname
+  _sharedAskForUserData._checkLoggedIn(name)
```

Comment:
This is kind of confusing to me, we could be passing in a team name to check against the current username?

Reply:
this is us checking if our own username has changed (aka we logged in / out)

Reply:
oh this got mutated, needs to be state.config.username

Reply:
K that makes more sense, thanks!

Reply:
updated tnx!